keyboards/evyd13/atom47/keymaps/PHSC138/config.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
Resolved here [5c04134](https://github.com/qmk/qmk_firmware/pull/18768/commits/5c041345e76837e83a89a2d732937392c0d1e8e2)